src/components/menu/menu-gestures.ts

```diff
@@ -57,14 +57,28 @@ export class MenuContentGesture extends SlideEdgeGesture {
   }
 
   onSlide(slide: SlideData, ev: any) {
-    let z = (this.menu.side === 'right' ? slide.min : slide.max);
+    if (this.menu.side === 'right' || (this.menu.side === 'start' && this.plt.isRTL()) || (this.menu.side === 'end' && !this.plt.isRTL())) {
+      let z = slide.min;
+    }
+
```

Comment:
2 unnecessary break lines, it should be `} else {`.
Also, use `else` instead of `else if` as this is the exact opposite, and there is no default set.
This applies to lines: 62-64, 107-108
and on `menu-types` file as well: 89-91, 119-121, 153-154,
and on `menu` file:  344-345